Client/multiplayer_sa/CMultiplayerSA_Rendering.cpp

```diff
@@ -703,6 +715,30 @@ void _declspec(naked) HOOK_CVisibilityPlugins_RenderPedCB()
     }
 }
 
+// Hook info
+#define HOOKPOS_CRenderer_EverythingBarRoads                         0x553C78
+#define HOOKSIZE_CRenderer_EverythingBarRoads                        5
+DWORD RETURN_CRenderer_EverythingBarRoads = 0x553C7D;
+DWORD DO_CRenderer_EverythingBarRoads = 0x7EE180;
+void _declspec(naked) HOOK_CRenderer_EverythingBarRoads()
```

Comment:
This might not work that well for objects with alpha.
Have you tried that?

Reply:
You're right, this won't work well for translucent objects. Obviously, translucent objects require a dedicated render path involving the batch sorting. I'll fix it, okay.

Reply:
I think for now it's fine for it to not work, it can be fixed later.